storage/range_test.go

```diff
@@ -134,7 +134,13 @@ func (tc *testContext) Start(t *testing.T) {
 	tc.stopper.AddCloser(tc.transport)
 
 	if tc.store == nil {
-		tc.store = NewStore(tc.clock, tc.engine, nil, tc.gossip, tc.transport, TestStoreConfig)
+		ctx := TestStoreContext
+		ctx.Clock = tc.clock
+		ctx.Gossip = tc.gossip
+		ctx.Transport = tc.transport
+		// Dummy DB.
+		ctx.DB = &client.KV{}
```

Comment:
My concern is actually that a non-nil `&client.KV{}` might *not* explode, while a nil `KV` would fail more reliably. Looking at the struct, I'm not too concerned (an empty KV will have a nil Sender, which won't work), but I'd rather have nil here than a value that is not nil but still garbage.